src/node_util.cc

```diff
@@ -141,6 +141,12 @@ void Initialize(Local<Object> target,
   }
 #undef V
 
+  target->DefineOwnProperty(
+    env->context(),
+    OneByteString(env->isolate(), "pushValToArrayMax"),
+    Integer::NewFromUnsigned(env->isolate(), NODE_PUSH_VAL_TO_ARRAY_MAX),
+    v8::ReadOnly).FromJust();
```

Comment:
I assume the old `PropertyAttribute`-based API is used rather than the ES5 `PropertyDescriptor`-based one for ease of backporting?